packages/client/src/components/interface/Navigation.tsx

```diff
@@ -983,12 +978,14 @@ const mapStateToProps: (state: IStoreState) => IStateProps = (state) => {
     storedDeclarations: state.declarationsState.declarations,
     userDetails: getUserDetails(state),
     advancedSearchParams: getAdvancedSearchParamsState(state),
-    activeMenuItem: window.location.href.endsWith(WORKQUEUE_TABS.performance)
+    activeMenuItem: window.location.href.includes(WORKQUEUE_TABS.performance)
```

Comment:
@jpye-finch This will most likely break the sms notification & form configuration navigation for a national sys admin